adafruit_mcp230xx/mcp23017.py

```diff
@@ -234,3 +237,43 @@ def io_control(self):
     @io_control.setter
     def io_control(self, val):
         self._write_u8(_MCP23017_IOCON, val)
+
+    @property
+    def int_flg(self):
+        """Returns a list with the pin numbers that caused an interrupt
+        port A ----> pins 0-7
+        port B ----> pins 8-15
+        """
+        intf = self._read_u16le(_MCP23017_INTFA)
+        flg = [i for i in range(16) if intf & (1 << i)]
+        return flg
+
+    @property
+    def int_flga(self):
+        """Returns a list of pin numbers that coused an interrupt in port A
+        pins: 0-7
+        """
+        intfa = self._read_u8(_MCP23017_INTFA)
+        flga = [i for i in range(8) if intfa & (1 << i)]
+        return flga
+
+    @property
+    def int_flgb(self):
```

Comment:
Same here

Reply:
@makermelissa, Sorry it took me this long to respond, I couldn't get to this until today. I made the suggested changes and I also added an example that uses the new functions and the RPi.GPIO.add_event_detect function for an efficient use of the hardware interrupt functionality of the MCP23017 IC.